devel/qri/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        qri-io qri 0.9.3 v
+
+categories          devel net
+license             GPL-3
+platforms           darwin
+
+maintainers         {gmail.com:herby.gillot @herbygillot} openmaintainer
+
+checksums           rmd160  33f399e9856177d43dc7f10d8b24295b13b09389 \
+                    sha256  9207e22b5a47e1f258e591d81d403d983d028d81dac6d7120c545022346a1302 \
+                    size    4465272
+
+description         A global dataset version control system (GDVCS) built on \
+                    the distributed web.
+
+long_description    Qri is a global dataset version control system (GDVCS) \
+                    built on the distributed web. Qri is global, specific to \
+                    datasets, provides version control and runs on the \
+                    distributed web, seeking to  solve common data problems \
+                    around discovery, trust, friction & synchronization.
+
+homepage            https://qri.io/
+
+depends_build       port:go
+build.env           GO111MODULE=on
+build.target        build
```

Comment:
Is there a reason you are not using the golang portgroup?

Reply:
Yes, so that I could build using the project's Makefile